pkg/metallb/frrnodestate_test.go

```diff
@@ -0,0 +1,174 @@
+package metallb
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	defaultNodeName           = "worker-0"
+	defaultFrrNodeStateNsName = "test-namespace"
+)
+
+func TestFrrNodeStateGet(t *testing.T) {
+	testCases := []struct {
+		testFrrNodeState *FrrNodeStateBuilder
+		expectedError    error
+	}{
+		{
+			testFrrNodeState: buildValidFrrNodeStateTestBuilder(buildTestFrrClientWithDummyNode()),
+			expectedError:    nil,
+		},
+	}
+
+	for _, testCase := range testCases {
+		frrNodeState, err := testCase.testFrrNodeState.Get()
+		assert.Equal(t, testCase.expectedError, err)
+
+		if testCase.expectedError == nil {
+			assert.Equal(t, frrNodeState.Name, testCase.testFrrNodeState.Definition.Name)
+		}
+	}
+}
+
+func TestFrrNodeStateExist(t *testing.T) {
+	testCases := []struct {
+		testFrrNodeState *FrrNodeStateBuilder
+		nodes            []*FrrNodeStateBuilder
+		expectedStatus   bool
+	}{
+		{
+			testFrrNodeState: buildValidFrrNodeStateTestBuilder(buildTestFrrClientWithDummyNode()),
+			expectedStatus:   true,
+		},
```

Comment:
Make sure to add the test where it does not exist, [mcp_test.go example](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/mco/mcp_test.go#L233-L236)

Reply:
Added